cody-jones-imprivata/CsNades
# created a 404 page

Description of PR that completes issue here...

 Changes 

- Made 404.html,css,js page 
- Brings user to 404 screen when something goes wrong

 Testing 

1. run `git fetch`
2. run `serve`
3. go to `/pages/404page` and check if 404 page loads
4. check if coin flip works properly
5. check if link to home page works
